WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -148,7 +148,7 @@ class AccountSettingsService {
         // This was the simplest implementation. If performance is an issue, we could try
         // adding `distinctUntilChanged` or `filter` on the notification userInfo and only
         // emit if the changed objects include the observed account.
-        return notificationObserver.map(getSettings).startWith(getSettings())
+        return Observable.just(getSettings()).concat(notificationObserver.map(getSettings))
```

Comment:
This still has the same bug, as it's just a different way to write the same thing. What we want for that initial value is to call `getSettings` when the observable is subscribed to, not when it's created. I think the simplest way is to use `deferred`. 

I'm pushing ff9c412 with a fix, and hopefully adding a bit more clarity to the whole method.